pkgs/tools/networking/curl/default.nix

```diff
@@ -41,18 +41,8 @@ stdenv.mkDerivation rec {
 
   patches = [
     # remove these two patches for cURL >= 7.71.0
-    (fetchurl {
-      # https://www.openwall.com/lists/oss-security/2020/06/24/1
```

Comment:
I think we should keep the links to oss-security.

Reply:
Done :+1: